language/move-vm/runtime/src/move_vm.rs

```diff
@@ -28,6 +27,7 @@ impl MoveVM {
         function_name: &IdentStr,
         ty_args: Vec<TypeTag>,
         args: Vec<Value>,
+        _sender: AccountAddress,
```

Comment:
Any reason not to remove the arg?

Reply:
good question, I am not sure how this whole sender story will end up. Point being that not having the sender everywhere is odd. You can imagine that the semantic is that the client is always responsible for the sender and only when the API takes a sender that is materialized. You could imagine prologue and epilogue or other system API may want a sender. 
So I added for consistency and to review soon with the team just so we think about it and come to a closure